frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/task-sidebar/TaskSidebarName.tsx

```diff
@@ -46,7 +47,7 @@ const TaskSidebarName: React.FC<TaskSidebarNameProps> = (props) => {
           }
 
           setValidating(true);
-          nameValidationSchema(t, 63)
+          nameValidationSchema((tKey) => i18n.t(tKey), 63)
```

Comment:
Eg of a half-way change. If this is undesirable, I can make the change to the shared folder and in the 4-5 other packages it impacts.